pkg/sql/opt/memo/testdata/stats_quality/tpch/q06

```diff
@@ -43,17 +43,17 @@ scalar-group-by
  │    ├── save-table-name: q6_project_2
  │    ├── columns: column19:19(float!null)
  │    ├── immutable
- │    ├── stats: [rows=71117.89, distinct(19)=71117.9, null(19)=0]
+ │    ├── stats: [rows=173775.9, distinct(19)=173776, null(19)=0]
  │    ├── select
  │    │    ├── save-table-name: q6_select_3
  │    │    ├── columns: l_quantity:5(float!null) l_extendedprice:6(float!null) l_discount:7(float!null) l_shipdate:11(date!null)
- │    │    ├── stats: [rows=71117.89, distinct(5)=23.5306, null(5)=0, distinct(6)=68848.6, null(6)=0, distinct(7)=1.8, null(7)=0, distinct(11)=365, null(11)=0, distinct(6,7)=71117.9, null(6,7)=0, distinct(5,7,11)=15459.6, null(5,7,11)=0]
- │    │    │   histogram(5)=  0 2860.5 68257          0
- │    │    │                <--- 1.0 -------- 23.999999999999996
- │    │    │   histogram(7)=  0           0            71118   0
- │    │    │                <--- 0.049999999999999996 ------- 0.07
- │    │    │   histogram(11)=  0       0        923.01     138.42     2123     323.06     2076.7     415.35     2215.2     230.78     2123     184.56     1846     415.35     1892.2     415.35     2169.1     276.92     2030.6     230.78     2076.7     184.56     2215.2     276.92     2169.1     138.42     2215.2     184.56     2215.2     138.42     2169.1     138.42     2123     276.92     2169.1     230.78     2030.6     230.78     2169.1     92.282     2169.1     184.56     2215.2     92.282     2123     230.78     1892.2     415.35     2215.2     138.42     1984.5     276.92     2076.7     184.56     2215.2     138.42     2215.2     46.141     2215.2     46.141     2123     276.92     2024.5     144.61
- │    │    │                 <--- '1993-12-31' -------- '1994-01-06' ------ '1994-01-20' -------- '1994-02-01' -------- '1994-02-14' ------ '1994-02-25' ------ '1994-03-11' -------- '1994-03-24' -------- '1994-04-07' -------- '1994-04-19' -------- '1994-05-03' -------- '1994-05-14' -------- '1994-05-24' -------- '1994-06-04' -------- '1994-06-14' -------- '1994-06-26' ------ '1994-07-06' -------- '1994-07-17' -------- '1994-08-01' -------- '1994-08-11' -------- '1994-08-25' -------- '1994-09-05' ------ '1994-09-16' -------- '1994-09-26' -------- '1994-10-07' -------- '1994-10-17' -------- '1994-10-27' -------- '1994-11-09' -------- '1994-11-22' -------- '1994-12-04' ------ '1994-12-16' -------- '1994-12-31'
+ │    │    ├── stats: [rows=173775.9, distinct(5)=23, null(5)=0, distinct(6)=160567, null(6)=0, distinct(7)=4.5, null(7)=0, distinct(11)=365, null(11)=0, distinct(6,7)=173776, null(6,7)=0, distinct(5,7,11)=37777.5, null(5,7,11)=0]
```

Comment:
This change is certainly suspicious. I haven't yet worked out what is causing it.